src/ducks/sep24WithdrawAsset.ts

```diff
@@ -0,0 +1,143 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { RootState } from "config/store";
+import { accountSelector } from "ducks/account";
+import { settingsSelector } from "ducks/settings";
+import { getNetworkConfig } from "helpers/getNetworkConfig";
+import { log } from "helpers/log";
+import {
+  sep10AuthStart,
+  sep10AuthSign,
+  sep10AuthSend,
+} from "methods/sep10Auth";
+import {
+  checkToml,
+  checkInfo,
+  interactiveWithdrawFlow,
+  createPopup,
+  pollWithdrawUntilComplete,
+} from "methods/sep24";
+import {
+  ActionStatus,
+  RejectMessage,
+  Sep24WithdrawAssetInitialState,
+} from "types/types.d";
+
+export const withdrawAssetAction = createAsyncThunk<
+  { currentStatus: string },
+  { assetCode: string; assetIssuer: string },
+  { rejectValue: RejectMessage; state: RootState }
+>(
+  "sep24WithdrawAsset/withdrawAssetAction",
+  async ({ assetCode, assetIssuer }, { rejectWithValue, getState }) => {
+    const { data, secretKey } = accountSelector(getState());
+    const { pubnet } = settingsSelector(getState());
+    const networkConfig = getNetworkConfig(pubnet);
+    const publicKey = data?.id;
+
+    if (!publicKey) {
+      throw new Error("Something is wrong with Account, no public key.");
+    }
+
+    try {
+      // Check toml
+      const tomlResponse = await checkToml({
+        assetIssuer,
+        networkUrl: networkConfig.url,
+      });
+
+      // Check info
+      await checkInfo({ toml: tomlResponse, assetCode });
```

Comment:
`checkInfo()` checks if the asset is enabled for deposit. We should add a parameter to this function that specifies if we're checking for deposit or withdraw.

Reply:
Fixed in https://github.com/stellar/stellar-demo-wallet/pull/125